internal/binary.bzl

```diff
@@ -26,8 +26,8 @@ load(":run.bzl", "postcss_run")
 def postcss_binary(
```

Comment:
This still says that plugins' keys are require paths, but now they're deps.

Reply:
Done.